pkg/commands/internal/migrate/ptr/ptr.go

```diff
@@ -0,0 +1,12 @@
+package ptr
+
+func Deref[T any](v *T) T {
+	if v == nil {
+		var zero T
+		return zero
+	}
+
+	return *v
+}
+
+func Pointer[T any](v T) *T { return &v }
```

Comment:
It's debatable, but do you consider `Of` as an alternative name? Like https://pkg.go.dev/k8s.io/utils/ptr#To

It's also commonly used: https://sourcegraph.com/search?q=context:global+ptr.Of+lang:go+&patternType=standard&sm=1

Reply:
I prefer an explicit name.